app/controllers/api/v2_alpha/notifications_controller.rb

```diff
@@ -33,8 +33,8 @@ def index
         'app.notification_grouping.status.unique_count' => statuses.uniq.size
```

Comment:
We probably need to update this with the partial accounts? And maybe add the value of the `expand_accounts` attribute so we can compare the performance between the 2?